cfme/common/provider.py

```diff
@@ -310,22 +310,18 @@ def _fill_default_endpoint_dicts(self, provider_attributes, connection_configs):
                 "auth_type": "default",
                 "auth_key": endpoint_default.credentials.service_account,
             }
-            connection_configs.append(default_connection)
         else:
             raise AssertionError("Provider wasn't added. "
                 "No credentials info found for provider {}.".format(self.name))
 
-        cert = getattr(endpoint_default, "ca_certs", None)
-        if cert and self.appliance.version >= "5.8":
-            default_connection["endpoint"]["certificate_authority"] = cert
-            connection_configs.append(default_connection)
+        if getattr(endpoint_default, "ca_certs", None):
```

Comment:
Optional: as far as I can tell `getattr` is being used in the same was as `hasattr` throughout this file, so why not remain consistent?

Reply:
I think a better option is `hasattr` as we are just checking.